types/node/base.d.ts

```diff
@@ -18,3 +18,4 @@
 
 // TypeScript 3.7-specific augmentations:
 /// <reference path="assert.d.ts" />
+/// <reference path="assert/strict.d.ts" />
```

Comment:
This should be moved to `ts3.4/base.d.ts`, since `assert/strict.d.ts` doesn’t differ between **TypeScript ≥3.7** and **TypeScript ≤3.6**.